examples/network_profile/README.md

```diff
@@ -7,3 +7,7 @@ This directory covers examples on how to create different kinds of Network Profi
 Follow these examples for setting up Network Profiles:
 
 * Setup [network_profile](../network_profile/simple/README.md) with no isolation
```

Comment:
Oops, perhaps I introduced this error. The relative path from this file to child directories should not include "../network_profile". This pertains to all of these new links.

Reply:
You are right. They are not necessary. But not breaking the navigation as well. I removed the prefix from all the links this file.